### PR TITLE
Feat/video full toggle

### PR DESCRIPTION
**Pull request by TurnbulA**
Used js to create full screen toggle 
Using font awesome for the icon

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,10 +73,21 @@ <h1 class="header-title-text">
         </div>
       </div>
       <div class="header-image-box">
-        <img class="header-image" src="https://picsum.photos/1000/500" />
-        <div class="button-border">
-          <div class="play"></div>
+        <video id="video-player" muted loop>
+          <source
+            src="resources/Video/Slaps Slow Motion.mp4"
```
Comment (mali891) [inline index.html]: change url to lowercase and replace spaces with hyphens

File: index.html [hunk 2]
```diff
@@ -73,10 +73,21 @@ <h1 class="header-title-text">
         </div>
       </div>
       <div class="header-image-box">
-        <img class="header-image" src="https://picsum.photos/1000/500" />
-        <div class="button-border">
-          <div class="play"></div>
+        <video id="video-player" muted loop>
```
Comment (mali891) [inline index.html]: Change to class, only use IDs if absolutely necessary

File: functions.js [hunk 3]
```diff
@@ -10,3 +10,22 @@ window.addEventListener("scroll", () => {
     ? nav.classList.add("c-nav--alt")
     : nav.classList.remove("c-nav--alt");
 });
+
+const pageVideo = document.getElementById("video-player");
+
+function playPause() {
+  if (pageVideo.paused) {
```
Comment (mali891) [inline functions.js]: ternary?

File: index.html (hunk 1 above)
Comment (TurnbulA) [reply index.html]: Done

**Review by mali891 — APPROVED** [review]

**Comment by mali891** [comment]
Looking good @TurnbulA

**Pull request merged**